CHANGELOG.md

```diff
@@ -1,6 +1,49 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+
+## [2.84.0]
```

Comment:
@vikasg-bb
Please update the json to yaml format, only modelbank is using json format explicitly in our modelbank bootstrap task otherwise rest or default is yaml format of LE hierarchy.

second rather than just mentioning contact structure also mention other levels where this contact needs to be added. refer limits example in this chagelog file.